include/deal.II/grid/grid_tools.h

```diff
@@ -468,20 +468,27 @@ namespace GridTools
   /*@{*/
 
   /**
-   * Find and return the number of the used vertex in a given mesh that is
-   * located closest to a given point.
+   * Find and return the number of the used vertex (or marked vertex) in a
+   * given mesh that is located closest to a given point.
    *
    * @param mesh A variable of a type that satisfies the requirements of the
    * @ref ConceptMeshType "MeshType concept".
    * @param p The point for which we want to find the closest vertex.
+   * @param[Optional] marked_vertices An array of bools indicating whether an
+   * entry in the vertex array should be considered (and the others must
```

Comment:
maybe "...indicating which vertices of a Mesh object will be considered within the search"